HSchool-project/src/app/Course.ts

```diff
@@ -0,0 +1,7 @@
+export interface Course{
+    id: number;
+    courseName: string;
+    progress: number;
+    author: string;
+    courseNo: number;
+  }
```

Comment:
- [ ] This file should be a declaration file, I mean its name should be something like `Course.d.ts`